src/views/registration/attendees/loader.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * This template renders the attendee registration loader
+ *
+ * @since TBD
+ * @version TBD
+ *
+ */
+
+?>
+<div class="tribe-block__tickets__item__attendee__fields__loader">
+  <svg version="1.1" id="L4" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 100" enable-background="new 0 0 0 0" xml:space="preserve">
```

Comment:
Q: Any reason to not use `<?php echo file_get_contents("loader.svg"); ?>` other than an additional HTTP request (which could be enough justification)? Just thinking of reusable components (specifically the svg itself), and that approach should give you the same inline experience/control.

Reply:
Heck, add a filter for the file name and it allows customization for users as well...